### PR TITLE
[FLINK-3416] [py] Support for spaces in flink path

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

**Comment by zentol** [comment]
This PR adds "" around the flink path within python bat files, so that paths containing spaces are treated as a single argument.

File: flink-dist/src/main/flink-bin/bin/pyflink2.bat [hunk 1]
```diff
@@ -22,4 +22,4 @@ setlocal EnableDelayedExpansion
 SET bin=%~dp0
 SET FLINK_ROOT_DIR=%bin%..
 
-%FLINK_ROOT_DIR%\bin\flink run -v %FLINK_ROOT_DIR%\lib\flink-python*.jar 2 %*
\ No newline at end of file
+"%FLINK_ROOT_DIR%\bin\flink" run -v "%FLINK_ROOT_DIR%"\lib\flink-python*.jar 2 %*
```
Comment (greghogan) [inline flink-dist/src/main/flink-bin/bin/pyflink2.bat]: Does the wildcard argument also need double quotes?

Comment (zentol) [inline flink-dist/src/main/flink-bin/bin/pyflink2.bat]: no, the jar name will never contain spaces

**Comment by tillrohrmann** [comment]
Changes look good to me. Thanks for your contribution @zentol. Will merge it.

File: flink-dist/src/main/flink-bin/bin/pyflink2.bat (hunk 1 above)
Comment (greghogan) [inline flink-dist/src/main/flink-bin/bin/pyflink2.bat]: I was thinking of quotes around `%*`. This is necessary in the shell scripts.

If you haven't already, could you try something like WordCount with spaces in the output file name?

Comment (zentol) [inline flink-dist/src/main/flink-bin/bin/pyflink2.bat]: that would result in all arguments passed being treated as a single argument.

if additional paths are 
* relative, Flink will receive a relative path and use internal tools to make them absolute.
* absolute, the user has to use "" himself.

**Pull request closed without merging**